src/eslint/plugin.ts

```diff
@@ -1,14 +1,9 @@
 import { consistentReferenceElementDeclarations } from './rules/consistent-reference-element-declarations.js';
 import { consistentTestFixtureVariableDeclarator } from './rules/consistent-test-fixture-variable-declarator.js';
 import { noNestedTemplateLiterals } from './rules/no-nested-template-literals.js';
-import { noOnlyTests } from './rules/no-only-tests.js';
 import { noRedudantPropertyAttribute } from './rules/no-redundant-property-attribute.js';
 import { noRedudantPropertyStringType } from './rules/no-redundant-property-string-type.js';
-import { noSkipTests } from './rules/no-skip-tests.js';
-import { noSpacePress } from './rules/no-space-press.js';
-import { noToHaveAttribute } from './rules/no-to-have-attribute.js';
 import { preferClosedShadowRoot } from './rules/prefer-shadow-root-mode.js';
-import { betterTestAssertions } from './rules/better-test-assertions.js';
```

Comment:
Figured we could remove these rules now given we won't be writing any new Web Test Runner tests.